src/types/task.list.ts

```diff
@@ -0,0 +1,56 @@
+import { DateTime } from "luxon";
+import { readableFormat } from "../utils/date.formatter";
+
+export enum Task {
```

Comment:
The enum name doesn't look correct to me.  NOT_CHECKED, IN_PROGRESS, CHECKED are not types of Task, they are TaskState

Reply:
Updated the name